samcli/commands/_utils/template.py

```diff
@@ -161,6 +156,31 @@ def _update_relative_paths(template_dict, original_root, new_root):
     return template_dict
 
 
+def _transform_dict(template_dict, path, transformation, context=None):
```

Comment:
As you import this method in another file, it shouldn't be private.
Also you should add docstrings.

Reply:
You mean it's imported on the unit test? I besides file it's declared on, it should only be used in `test_template.py`. Can you link where else it's being imported?

Reply:
In `artifact_explorer.py`: https://github.com/aws/aws-sam-cli/pull/2422/files#diff-9d41d813918d5d2434b3b5397b8017579cfbe1c1f53b0f242959d6eee874346aR268

Reply:
nvm, lost it on the merging. found it